src/java/org/apache/cassandra/service/StorageService.java

```diff
@@ -626,7 +626,7 @@ public void stopTransports()
      * they get the Gossip shutdown message, so even if
      * we don't get time to broadcast this, it is not a problem.
      *
-     * See {@link Gossiper#markAsShutdown(InetAddressAndPort)}
+     * See Gossiper.markAsShutdown(InetAddressAndPort)
```

Comment:
Is this syntax correct?

Reply:
@bereng probably isnt but if you take a closer look, that method is not accessible from this class and it marks it as errorneous in IDEA (whole file) and it drives me crazy. 

As that method is not reachable from here, I would just completely remove that link, wdyt?